src/main/java/com/synopsys/integration/alert/provider/blackduck/BlackDuckValidator.java

```diff
@@ -43,15 +41,13 @@
 import com.synopsys.integration.log.IntLogger;
 import com.synopsys.integration.log.Slf4jIntLogger;
 
-@Component
 public class BlackDuckValidator extends ProviderValidator {
     private static final Logger logger = LoggerFactory.getLogger(BlackDuckValidator.class);
 
     private final AlertProperties alertProperties;
     private final BlackDuckProperties blackDuckProperties;
     private final SystemMessageUtility systemMessageUtility;
 
-    @Autowired
     public BlackDuckValidator(AlertProperties alertProperties, BlackDuckProperties blackDuckProperties, SystemMessageUtility systemMessageUtility) {
```

Comment:
Should the `validate()` method take a properties object rather than having a factory for it?

Reply:
I can make those changes.